src/ui/components/filters/facetscomponent.js

```diff
@@ -40,7 +40,7 @@ class FacetsConfig {
      * If true, show a "reset all" button to reset all facets
      * @type {boolean}
      */
-    this.resetFacets = config.resetFacets === undefined ? true : config.resetFacets;
```

Comment:
README change please

Reply:
also sounds like we need a README change for resetFilters in FilterBox?

Reply:
resetFilters README was already updated, will update facets README

Reply:
sorry coulda swore I pushed this, updated!